src/lab_sim/objectives/pick_and_place_example.xml

```diff
@@ -22,10 +23,14 @@
           controller_names="/joint_trajectory_controller /robotiq_gripper_controller"
           task="{mtc_pick_task}"
         />
-        <Action ID="SetupMTCCurrentState" task="{mtc_pick_task}" />
-        <!-- The following behavior can be found in the example_behaviors package. -->
         <Action
-          ID="SetupMtcPickFromPose"
+          ID="SetupMTCCurrentState"
+          task="{mtc_pick_task}"
+          skip_collision_check="false"
+        />
+        <!--The following behavior can be found in the example_behaviors package.-->
+        <Action
+          ID="ExampleSetupMtcPickFromPose"
```

Comment:
Unless we remove the COLCON_IGNORE, I don't like having an objective that cannot be run in the examples.

In the past, I've done something like
```suggestion
          # An example implementation of this behavior called ExampleSetupMtcPickFromPose can be found in the example_behaviors package
          ID="SetupMtcPickFromPose"
```